docs/output.md

```diff
@@ -606,6 +584,47 @@ As described in the documentation, [ASCIIGenome](https://asciigenome.readthedocs
 
 <p align="center"><img src="images/asciigenome_screenshot.png" alt="ASCIIGenome screenshot"></p>
 
+### iVar consensus
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `variants/<CALLER>/consensus/ivar/`
+    * `*.consensus.fa`: Consensus Fasta file generated by iVar.
+    * `*.consensus.qual.txt`: File with the average quality of each base in the consensus sequence.
+* `variants/<CALLER>/consensus/ivar/base_qc/`
+    * `*.ACTG_density.pdf`: Plot showing density of ACGT bases within the consensus sequence.
+    * `*.base_counts.pdf`: Plot showing frequency and percentages of all bases in consensus sequence.
+    * `*.base_counts.tsv`: File containing frequency and percentages of all bases in consensus sequence.
+    * `*.N_density.pdf`: Plot showing density of N bases within the consensus sequence.
+    * `*.N_run.tsv`: File containing start positions and width of N bases in consensus sequence.
+
+**NB:** The value of `<CALLER>` in the output directory name above is determined by the `--variant_caller` parameter (Default: 'ivar' for '--protocol amplicon' and 'bcftools' for '--protocol metagenomic').
+
+</details>
+
+As described in the [iVar variants](#ivar-variants) section, iVar can be used in this pipeline to call variants and for the consensus sequence generation.
+
+### BCFTools and BEDTools
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `variants/<CALLER>/consensus/bcftools/`
+    * `*.consensus.fa`: Consensus Fasta file generated by integrating the variants called by BCFTools into the reference genome.
```

Comment:
Shouldn't it be the variants called by <CALLER> it could be bcftools or ivar right?
Also another output here would be the variants with AF> threshold?

Reply:
Good spot. Fixed in https://github.com/nf-core/viralrecon/pull/265/commits/e48bef735e857cb8d7a3756330cea30a263cc3e0

Yep, the variants with AF > threshold are defined in the line below that one. They will be called `<SAMPLE>.filtered.vcf.gz`